UPGRADING.md

```diff
@@ -221,7 +228,11 @@ From here on the doc will be focussed on an ODFE plugin as an example.
 
 See [anomaly-detection#35](https://github.com/opensearch-project/anomaly-detection/pull/35) for an example.
 
-### OpenSearch Dashboard Plugins
+##### Indices
+
+Do not change index names at this time to preserve backwards compatibility.
```

Comment:
All of the regular indices will work good. 
I think we should call out on a path for .kibana index for plugins.

Reply:
Let's leave a section on dashboards for next PR. Maybe someone from Dashboards, e.g. @mihirsoni can write those?